tests/projects/external-kit/Twoliter.toml

```
@@ -0,0 +1,7 @@
+schema-version = 1
+release-version = "1.0.0"
+
+[sdk]
+registry = "twoliter.alpha"
+repo = "bottlerocket-sdk"
+tag = "latest"
```

thought we were going to align this to:
```suggestion
[sdk]
name = "bottlerocket-sdk"
version = "1.4.2"
vendor = "bottlerocket"

[vendor.bottlerocket]
registry = "..."
```

We should clean that up as a preliminary commit / PR change so as not to crowd this one. I'm also fine if we commit the unit tests first.

---

The changes to Twoliter.toml is coming in another PR I will put that one up today